sample.html

```diff
@@ -5,8 +5,9 @@
   <script src="//player.ooyala.com/static/v4/production/latest/core.min.js"></script>
   <script src="//player.ooyala.com/static/v4/production/latest/video-plugin/main_html5.min.js"></script>
   <script src="//player.ooyala.com/static/v4/production/latest/video-plugin/bit_wrapper.min.js"></script>
-  <script src="//player.ooyala.com/static/v4/production/latest/skin-plugin/html5-skin.min.js"></script>
-  <link rel="stylesheet" href="//player.ooyala.com/static/v4/production/latest/skin-plugin/html5-skin.min.css"/>
+  <!--<script src="//player.ooyala.com/static/v4/production/latest/skin-plugin/html5-skin.min.js"></script>-->
```

Comment:
What was this change done for?

Reply:
Html-webpack-plugin uses this file as a template to inject there the generated `html5-skin.min.js` and `html5-skin.min.css`. So with webpack-dev-server we will see the actual changes during development. 
I commented those `script` and `link` tags to avoid duplications in that case.
Is it better to revert changes in this file and create a new file - template for html-webpack-pluginwithout those lines ?

Reply:
I think it would be better to do that. This sample.html is meant to show users of our skin a sample of how things are used.